recipe/meta.yaml

```
@@ -50,7 +50,9 @@ requirements:
     - r-magrittr
     - r-s2 >=1.1.0
     - r-units >=0.7_0
-    - libgdal >=3.6.0
+    - libgdal >=3.6.2
+    - geos >=3.8.0
+    - proj >=9.3.1
```

```suggestion
    - libgdal {{ libgdal }} 
    - geos {{ geos }}
    - proj {{ proj }}
    - sqlite {{ sqlite }}
```
Best practice is to pin to the lowest bound of what we want to support.